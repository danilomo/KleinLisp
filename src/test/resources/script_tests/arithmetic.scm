(println (+ 1 2 3))
(println (- 10 3))
(println (* 4 5))
(println (/ 15 3))
(println (+ (* 2 3) (- 8 2)))