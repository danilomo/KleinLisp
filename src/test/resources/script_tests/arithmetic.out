6
7
20
5
12